.github/workflows/main.yml

```diff
@@ -91,7 +91,7 @@ jobs:
           python-version: '3.8'
           cache: 'pip'
           cache-dependency-path: requirements-dev.txt
-
+  
```

Comment:
Try not to introduce unnecessary whitespace changes. Something like `git diff master` would reveal these before pushing.